nucypher/blockchain/eth/sol/source/contracts/WorkLock.sol

```diff
@@ -157,26 +211,163 @@ contract WorkLock {
     * @notice Cancel bid and refund deposited ETH
     */
     function cancelBid() external {
-        // TODO check date? check minimum amount of tokens? (#1508)
+        require(block.timestamp < endCancellationDate, "Cancellation allowed only during bidding");
```

Comment:
The cancellation period can extend to after the `endBidDate` no? In which case, it's not really during 
"bidding" but rather until {endCancellationDate} - or is 
"bidding" the term for everything before claim?

Reply:
This is simple: I forgot to change
I did some refinements to keep cancellation window as bidding + some after time